precovery/frame_db.py

```diff
@@ -820,12 +828,13 @@ def defragment(self, new_index: FrameIndex, new_db: "FrameDB"):
                 )
                 new_frame = HealpixFrame(
                     id=None,
-                    obscode=cur_key[0],
-                    filter=cur_key[1],
-                    exposure_mjd_start=cur_key[2],
-                    exposure_mjd_mid=cur_key[3],
-                    exposure_duration=cur_key[4],
-                    healpixel=cur_key[5],
+                    dataset_id=cur_key[0],
```

Comment:
That seems like a good idea to me, yes. I think any performance impact should be totally negligible and it would be a lot more readable.

Reply:
It turns out this function was pretty dramatically wrong, since it depended on sorting in another query, which also wasn't updated. I'm making a separate PR to fix that.